src/send_bw.c

```diff
@@ -378,9 +378,8 @@ int main(int argc, char *argv[])
 	}
 
 	if (user_param.test_method == RUN_ALL) {
-
-		if (user_param.connection_type == UD)
-			size_max_pow =  (int)UD_MSG_2_EXP(MTU_SIZE(user_param.curr_mtu)) + 1;
+		size_max_pow = MSG_SZ_2_EXP(user_param.size) + 1;
```

Comment:
Why is it needed?

Reply:
Previously, the BW tests with RUN_ALL ran all sizes 2^(1...24).
SRD does not support sending messages this big, and it is my understanding that other connections won't be able to send messages if the size is greater than what's reported in max message size in port attributes.

Reply:
Let's not change the behavior and put a condition here for SRD

Reply:
Ack.